tests/unit/amo/sagas/test_home.js

```diff
@@ -421,6 +444,41 @@ describe(__filename, () => {
       expect(expectedAction).toEqual(errorAction);
     });
 
+    it('aborts fetching for a failed search fetch', async () => {
+      const state = sagaTester.getState();
+
+      const slug = 'collection-slug';
+      const userId = 123;
+
+      mockHeroApi
+        .expects('getHeroShelves')
+        .returns(Promise.resolve(createHeroShelves()));
+
+      const firstCollection = createFakeCollectionAddonsListResponse();
+      mockCollectionsApi
+        .expects('getCollectionAddons')
+        .withArgs({
```

Comment:
The arguments are already covered by other tests so you can remove this line. Also, `mockCollectionsApi.verify()` is never called (which is fine) so this doesn't have an affect.